R/screenshot.R

```diff
@@ -0,0 +1,60 @@
+#' A function to make screenshots from an OTTR bookdown website
+#' @description This function creates screenshots of course chapters that are stored in a created output directory 
+#' @param git_pat default is NULL; required argument; a Git secret
+#' @param repo default is NULL; required argument; GitHub repository name, e.g., jhudsl/OTTR_Template
```

Comment:
What's the best way to communicate that these are required arguments, but by default NULL?

Reply:
I would just not have them be NULL if they are required because then it will automatically throw an error if no argument is provided. And docs wise the way you say its "required" is fine :+1: